source/installation-guide.rst

```diff
@@ -147,186 +162,52 @@ instance: ::
-that is accessible via SSH or event your local machine if it can reach
-the kubernetes cluster with a configuration file.
-
-On the example file, the access is done via a local node, where
-the ansible script is executed. This node is described as localhost
-in the hosts item of the group **all**.
-
-These same nodes must be added as children of the group dojot-k8s.
-
-To configure a local access on the hosts file, follow the example below:
-
-.. code:: yaml
-
-  ---
-  all:
-    hosts:
-      localhost:
-        ansible_connection: local
-        ansible_python.version.major: 3
-    children:
-      dojot-k8s:
-        hosts:
-          localhost:
-
-To configure remote access via ssh to a node of the cluster, follow
-this other example:
-
-.. code:: yaml
-
-  ---
-  all:
-    hosts:
-      NODE_NAME:
-        ansible_host: NODE_IP
-    children:
-      dojot-k8s:
-        hosts:
-          NODE_NAME:
-
-The next step is configuring the mandatory and optional variables
-required for deploying dojot.
-
-There is a document describing each of the variables that can be
-configured at `Ansible dojot variables`_.
-
-This variables must be set for the group '*dojot-k8s*', to do so set
-their values on the file dojot.yaml on the folder '**group_vars/dojot-k8s/**'
-
-.. _Ansible dojot: https://github.com/dojot/ansible-dojot
-.. _Ansible dojot variables: https://github.com/dojot/ansible-dojot/blob/master/docs/vars.md
+.. note::
 
-4. Executing the deployment playbook
-....................................
+  If you drop the dojot volumes you will also lose all data that you have collected on the platform
+  so far.
 
-Now that the inventory is set, the next step is executing
-the deployment playbook.
+To drop the volumes just pass the '-v' parameter in the 'docker-compose down' command as
+displayed below: ::
 
-To do so, run the following command:
+  docker-compose down -v
 
-.. code:: bash
+That way volumes and dojot will be dropped and you will be able to deploy a different dojot version.
 
-  ansible-playbook -K -k -i inventories/YOUR_INVENTORY deploy.yaml
+Usage
+^^^^^
 
-Wait for the playbook execution to finish without errors.
+The web interface is available at ``http://localhost:8000``. The user is
+``admin`` and the password is ``admin``. You also can interact with platform
+using the :doc:`./components-and-apis`.
 
-5. Accessing the deployed dojot environment
-...........................................
+Read the :doc:`using-api-interface` and :doc:`using-web-interface` for more
+information about how to interact with the platform.
 
-Dojot access will be set using NodePorts, to view the proper ports to access
-the environment it is necessary to check service configuration.
+Kubernetes
+----------
 
-.. code:: bash
+For installation with kubernetes please check the pdf below.
 
-  kubectl get service -n dojot kong iotagent-mosca
+:download:`click here to access the dojot installation guide with kubernetes <pdf/Dojot-Installation-Guide.pdf>`
```

Comment:
Should kubernetes start with a capital letter? (**K**ubernetes)
There are places that are in small letters and others that are in capital letters, it would be interesting to keep everything consistent.